pgjdbc/src/main/java/org/postgresql/jdbc/PgResultSet.java

```diff
@@ -2098,9 +2098,6 @@ public long getLong(int columnIndex) throws SQLException {
     if (isBinary(columnIndex)) {
       int col = columnIndex - 1;
       int oid = fields[col].getOID();
-      if (oid == Oid.INT8) {
-        return ByteConverter.int8(this_row[col], 0);
```

Comment:
Fair enough, I'm not married to this change. More problematic is this test doesn't fail